src/views/blocks/tickets.php

```diff
@@ -17,14 +17,20 @@
  * @var Tribe__Tickets__Editor__Template $this
  */
 
-$cart_classes        = [ 'tribe-block', 'tribe-block__tickets' ];
+
+/** @var Tribe__Tickets__Commerce__Currency $currency */
+$currency        = tribe( 'tickets.commerce.currency' );
+
+$cart_classes        = [ 'tribe-block', 'tribe-block__tickets', 'tribe-common' ];
 $cart_url            = $this->get( 'cart_url' );
 $has_tickets_on_sale = $this->get( 'has_tickets_on_sale' );
 $is_sale_past        = $this->get( 'is_sale_past' );
 $provider            = $this->get( 'provider' );
 $provider_id         = $this->get( 'provider_id' );
 $tickets             = $this->get( 'tickets', [] );
 $tickets_on_sale     = $this->get( 'tickets_on_sale' );
+// We're assuming that all the currency is the same here.
+$currency_symbol     = $currency->get_currency_symbol( $tickets[0]->ID, true );
```

Comment:
This runs before we check if there are tickets. Can we run this logic where we call this view instead?